config/initializers/valid_resource.rb

```diff
@@ -1,20 +1,9 @@
 class ValidResource
   BLACKLIST = {
-    article:     %w(about-our-debt-work am-ein-gwaith-dyled
-                    debt-publications cyhoeddiadau-ar-ddyledion
-                    partners-overview-parhub
```

Comment:
cool, there are some redirects in scripts, that we may need to double check that they are still needed, but :+1: